### PR TITLE
AML-1946-2107-Transfer-Invitation-Response-Notification

### PR DESCRIPTION
**Pull request by DScuffle**
Has 3 Email templates put onto Gov Notify as well.

Notification link has been left as comment deliberately we need a strategy for this navigation to specific pages.

**REQUIRES DEPLOYMENT OF das-Notifications with config build above 76931.**

### CODE REVIEW

File: src/SFA.DAS.EAS.MessageHandlers.Worker/EventHandlers/ApprovedTransferConnectionInvitationEventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        {
+            _db = dbContext;
+            _notificationsApi = notificationsApi;
+            _employerApprenticeshipsServiceConfiguration = employerApprenticeshipsServiceConfiguration;
+            _log = log;
+        }
+
+        protected override async Task ProcessMessage(ApprovedTransferConnectionInvitationEvent messageContent)
+        {
+            var users = _db.Users
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.MessageHandlers.Worker/EventHandlers/ApprovedTransferConnectionInvitationEventHandler.cs]: It looks like this query is used in a few places, could it be extracted out into an extension method hanging off `IQueryable<User>`?

File: src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.config [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.config]: Is this file needed?

File: src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.xsd [hunk 3]
```diff
@@ -0,0 +1,3103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (elliotritchie) [inline src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.xsd]: Is this file needed?

Comment (DScuffle) [reply src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.xsd]: Nope - Removing

File: src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.config (hunk 2 above)
Comment (DScuffle) [reply src/SFA.DAS.EAS.MessageHandlers.Worker/NLog.config]: Nope - Removing

File: src/SFA.DAS.EAS.MessageHandlers.Worker/EventHandlers/ApprovedTransferConnectionInvitationEventHandler.cs (hunk 1 above)
Comment (DScuffle) [reply src/SFA.DAS.EAS.MessageHandlers.Worker/EventHandlers/ApprovedTransferConnectionInvitationEventHandler.cs]: Agreed - extracted.
